src/main/java/mat/server/service/impl/MatContextServiceUtil.java

```diff
@@ -206,4 +228,11 @@ private BigDecimal asDecimalVersion(String version) {
         }
     }
 
+    public boolean isMeasureEditable(Measure measure) {
+        return featureFlag.findFeatureFlags().getOrDefault(FeatureFlagConstant.FHIR_EDIT, false) || MeasureDetailsUtil.QDM.equals(measure.getMeasureModel());

```

Comment:
1. I didn't notice that at first sight.. but.. it will make Pre-QDM measures editable too. Do we want that?
2. there are similar methods isEditable and isMeasureEditable which have different logic.. It can be confusing. How about we rename it into isMeasureModelEditable ?

Reply:
I don't see any Pre-QDM measures.

Reply:
You don't see it where? Ui has this logic, if model name is empty then it's pre cql/qdm

Reply:
yup, But server side doesn't know Pre-CQL. May be I should write test case for null values then.

Reply:
That what i meant